tests/ui/perfherder/graphs_view_test.jsx

```diff
@@ -368,8 +368,9 @@ describe('Mocked API calls', () => {
         subtests: 0,
       })}`,
       {
-        fcefb979eac44d057f9c05434580ce7845f4c2d6: {
+        '1647494': {
           id: 1647494,
+          signature_hash: 'fcefb979eac44d057f9c05434580ce7845f4c2d6',
```

Comment:
There can be a test with 2 different signatures that have the same signature_hash, if the test doen't require too mush setup to do

Reply:
Not sure there's any valuable test we can add here. This was mostly a backend bug & the frontend pretty much worked as expected.